dart/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright 2012, the Dart project authors. All rights reserved.
```

Comment:
Not sure if this file can be here. Google has a requirement for copyright assignment when doing a PR to a Google repo. Since this is from code originally from someone at Google, can we let this fall under the root Apache 2 license? Can you ask @scheglov or whoever you got the base code from?

Reply:
I guess if it comes from Chromium then it will be fine for Google. As far as being included in this folder, it could also be copied from the root by your script, if the same license can be used.